packages/suite/src/hooks/wallet/__tests__/useSendForm.test.tsx

```diff
@@ -214,9 +214,6 @@ const actionCallback = (
 };
 
 describe('useSendForm hook', () => {
-    beforeEach(() => {
```

Comment:
also in `useRbfForm.test` and `urls.test`

Reply:
[ba09764](https://github.com/trezor/trezor-suite/pull/8726/commits/ba09764746a54456e6ab1206b75ef2d79b924d73) urls is different package